<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyPlanWindow</class>
 <widget class="QMainWindow" name="StudyPlanWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>982</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 243, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonMenu">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>225</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/images/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>291</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(119, 102, 198);</string>
    </property>
    <property name="text">
     <string>PLAN DE ESTUDIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_degree">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(70, 70, 122);</string>
    </property>
    <property name="text">
     <string>Ingenier√≠a</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rectangle_background">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>458</x>
      <y>214</y>
      <width>966</width>
      <height>513</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 217, 217);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Semestre">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>901</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 250, 250);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_studyplan">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
